main/states/idle_state.py

```diff
@@ -43,11 +44,23 @@ def on_enter(self, payload=None):
         self.notify_renderer('idle')
 
     def __detected(self):
-        if self.isActive:
+        if hasattr(self, 'video_process') and self.video_process != None:
```

Comment:
You only check for a video process, but not for an audio process, please update this, too

Reply:
solved in the latest commit